web/frontend/src/types/overrides.ts

```diff
@@ -19,7 +19,7 @@ export interface MappingMetadata {
 }
 
 interface MappingDetails {
-  mode?: 'localize' | 'create' | 'update';
+  mode: 'localize' | 'create' | 'update' | 'undefined';
```

Comment:
All usage of this override was removed above. Why are we keeping it in the interface?